docs/getting-started/kubernetes-quickstart.md

```diff
@@ -0,0 +1,155 @@
+# Getting started with tracee in Kubernetes 
+
+This guide is focused on running tracee on Kubernetes. To demonstrate how to use Tracee we will use minikube, a distribution of kubernetes that can run on your local machine.
+
```

Comment:
"Alternatively, you could use most other Kubernetes clusters such as microk8s or from a cloud provider"